x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plan/logical/Pivot.java

```diff
@@ -47,7 +48,20 @@ public Pivot(Source source, LogicalPlan child, Expression column, List<NamedExpr
 
     @Override
     protected Pivot replaceChild(LogicalPlan newChild) {
-        return new Pivot(source(), newChild, column, values, aggregates);
+
+       java.util.function.Function<Expression, Expression> withQualifierNull = (Expression e)-> {
```

Comment:
I would extract this to a private static attribute.

Reply:
done